src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl service;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(474797L);
+        user.setLogin("userLogin47");
+        user.setPassword("password1111");
+        user.setAge(25);
+    }
+
+    @Test
+    void register_UserValid_Ok() {
+        StorageDao storageDao = new StorageDaoImpl();
+        assertNotNull(service.register(user));
+        assertNotNull(storageDao.get(user.getLogin()));
```

Comment:
While you are testing the valid case here, you are missing tests for edge cases of valid input, such as the minimum valid password length, and the exact age limit. Also, consider separating the assertions to check if the user was added to the storage into a different test case.